doc-700 (#1092)
spectrocloud/librarium: content/docs/08-user-management/2-palette-rbac/2-project-scope-roles-permissions.md

</Tabs.TabPane>
</Tabs>

# Virtual Cluster
--------------------------------

|Role Names   | Description  |
|---|---|

> |Virtual Cluster Admin |The Virtual Cluster Admin role is a closure of all the virtual cluster operations. It is an administrative privilege for the App resources |

Reviewer: ```suggestion
|Virtual Cluster Admin |Provides administrative privilege to perform all virtual cluster operations on App resources.|
```

> |Virtual Cluster Editor|The Virtual Cluster Editor role can perform edit operations related to a virtual cluster, but the user  is not able to create or delete a virtual cluster|

Reviewer: ```suggestion
|Virtual Cluster Editor|Allows the user to perform edit operations on a virtual cluster but not to create or delete a virtual cluster.|
```

> |Virtual Cluster Viewer|The Virtual Cluster Viewer will be able to view all the virtual cluster resources, but not privileged to make modifications|

Reviewer: ```suggestion
|Virtual Cluster Viewer|Allows the user to view all the virtual cluster resources but not to modify them.|
```
